include/volk/volk_common.h

```diff
@@ -4,7 +4,17 @@
 ////////////////////////////////////////////////////////////////////////
 // Cross-platform attribute macros
 ////////////////////////////////////////////////////////////////////////
-#if defined(__clang__) && (!defined(_MSC_VER))
```

Comment:
I thought we had this line due to some bug with clang and MSVC.  Obviously, it works with Clang, GCC and MSVC at the moment (I rely on CI here). And I assume it works with Apple-Clang. I just want to double-check.

Reply:
Yes your memory is correct! So was I did was move this macro first, so that if MSVC then those attributes are used.  Next up is Clang (and thus, !MSVC), because -some- clang (but not all) also define __GNUC__; then finally __GNUC__ & then the default if none of them. The reordering takes care of our prior issue wit MSVC, and reduces macro complexity.

Reply:
Great. then, this is resolved.